cargo-apk/README.md

```diff
@@ -87,6 +87,11 @@ icon = "@mipmap/ic_launcher"
 # Defaults to the compiled artifact's name.
 label = "Application Name"
 
+# Folder containing extra shared libraries intended to be dynamically loaded at runtime.
+# Files matching `libs_folder/${android_abi}/*.so` are added to the apk
+# according to the specified build_targets.
+runtime_libs = "path/to/libs_folder"
```

Comment:
This (`[package.metadata.android.application]`) is the wrong place, these docs should reside under `[package.metadata.android]`.